tests/test_butler.py

```diff
@@ -2083,23 +2083,6 @@ def testTransferUuidToUuid(self):
         # Setting id_gen_map should have no effect here
         self.assertButlerTransfers(id_gen_map={"random_data_2": DatasetIdGenEnum.DATAID_TYPE})
 
-    def testTransferIntToInt(self):
-        with self.assertWarns(FutureWarning):
-            self.create_butlers(
-                "lsst.daf.butler.registry.datasets.byDimensions.ByDimensionsDatasetRecordStorageManager",
-                "lsst.daf.butler.registry.datasets.byDimensions.ByDimensionsDatasetRecordStorageManager",
-            )
-        # int dataset ID only allows UNIQUE
-        self.assertButlerTransfers()
-
-    def testTransferIntToUuid(self):
```

Comment:
If we are allowing migration for the moment, shouldn't we still be testing int to UUID? Presumably the problem is that we can't any longer create a test repo from scratch? The only alternative is to have a pre-created repo for the test but it's likely not worth it.